<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BtnSignUp">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Up!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="LabelUsername">
      <property name="text">
       <string>username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EditUsername"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LabelPassword">
      <property name="text">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EditPassword"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="LabelConfirm">
      <property name="text">
       <string>retype password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="EditConfirm"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="indicator">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Indicator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
